cfgov/v1/models/sublanding_filterable_page.py

```diff
@@ -46,8 +49,22 @@ class ActivityLogPage(SublandingFilterablePage):
 
     objects = PageManager()
 
-    def base_query(self, hostname):
-        return AbstractFilterPage.objects.live_shared(hostname)
+    @classmethod
+    def eligible_categories(cls):
+        categories = dict(ref.categories)
+        return sorted(itertools.chain(*(
+            dict(categories[category]).keys()
+            for category in ('Blog', 'Newsroom', 'Research Report')
+        )))
```

Comment:
Since I'm not set up to test locally, I'm going to verify the obvious, since I saw there was some convo re this on #2558 -- these "parent" categories map to all the specific subcategories?